### ISSUE TITLE
Harmc val data missing.

### ISSUE THREAD

**Issue by singing-cat** [HKBUNLP/ExplainHM-WWW2024#8]
First of all, thank you for your great work and making data and code available to public. According to your repo, I think the **validation data of Harmc** is missing (more specifically, the llm debate and llm judge data), but 3013 training set and 354 test set data is fine (exists in your pkl file). Could you please offer the incomplete val data? Thank you very much!

**Comment by DanielLin97** [comment]
Thank you. The data is just used as an example for your reference but is outdated due to the update of emerging llms, and you can directly generate the new llm debate and llm judge data with more advanced llm. Thank you.
